converter/shieldhit/parser.py

```diff
@@ -209,10 +211,16 @@ def _get_scoring_filter_by_uuid(self, uuid: str) -> str:
 
     def _parse_geo_mat(self, json: dict) -> None:
         """Parses data from the input json into the geo_mat_config property"""
+        self._parse_title(json)
         self._parse_materials(json)
         self._parse_figures(json)
         self._parse_zones(json)
 
+    def _parse_title(self, json: dict) -> None:
+        """Parses data from the input json into the geo_mat_config property"""
+        if "title" in json["project"] and json["project"]["title"].length > 0:
```

Comment:
what kind of object is stored in `json["project"]["title"]`, so you are able to check its length using `length` attribute ? If that is a string or list I'd simply use `len` function